cmd/command.go

```diff
@@ -141,6 +156,7 @@ func New(config Config) (*cobra.Command, error) {
 	f.Init(m)
 
 	m.AddCommand(cl001Cmd)
+	m.AddCommand(cl002Cmd)
```

Comment:
I noticed that we do not generate the code in this file at all. Not sure right now if we should generate this file as well. We can talk about it. Just making it work now.

Reply:
thats fine